daemon/cmd/daemon_main.go

```diff
@@ -637,6 +637,9 @@ func initializeFlags() {
 	flags.Bool(option.EnableSessionAffinity, false, "Enable support for service session affinity")
 	option.BindEnv(option.EnableSessionAffinity)
 
+	flags.Bool(option.EnableServiceTopology, false, "Enable support for service topology aware hints")
```

Comment:
'topology aware hints' sounds weird, hits cannot be aware of something; instead, something like 'topology awareness hints' makes more sense, for example, but also a bit odd without much of a context. I think it's best to take out either 'aware/ness'  or 'hints'. So either 'service topology aware hints' or 'service topology awareness', see what I mean?

Reply:
I've reused the same name as it's named in K8s: https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/.

Reply:
I see! Well, if you ask me, Kubernetes is  not always the best standard ;)